src/Our.Umbraco.RobotsTxtEditor/Web/UI/App_Plugins/RobotsTxtEditor/views/robotstxteditor.html

```diff
@@ -0,0 +1,40 @@
+<div class="umb-robotstxteditor" ng-controller="RobotsTxtEditorController as vm">
+
+    <umb-box>
+        <umb-box-content>
+            
+            <h3 class="bold">Robots.txt Editor</h3>
+
+            <umb-load-indicator ng-show="vm.loading"></umb-load-indicator>
+
+            <div class="umb-robotstxteditor-main" ng-show="!vm.loading">
+                <div class="umb-robotstxteditor-controls">
+                    <p>
+                        <!-- TODO: replace with 'posh' textarea with line numbers (like editing styles)-->
+                        <textarea ng-model="vm.data.FileContents"></textarea>
+                    </p>
+                    <p>
+                        <umb-button type="button"
+                                    label="{{vm.data.FileExists ? 'Update Robots.txt' : 'Create Robots.txt'}}"
+                                    button-style="success"
+                                    action="vm.saveData()">
+                        </umb-button>
+
+                    </p>
+                </div>
+                <div class="umb-robotstxteditor-notes">
+                    <!-- TODO: add button to insert 'standard' rules for an Umbraco website -->
+                    <!-- TODO: add other buttons / helpful functionality -->
+                    <div ng-show="vm.data.FileExists">
+                        <umb-button type="button"
+                                    label="Reload from disk"
+                                    action="vm.reloadData()">

```

Comment:
🤔 Wondering if we need a Reload button. Are we expecting people to be editing it in both places at the same time? _and hoping that the person in the back-office will hit the reload button?_

Reply:
@leekelleher Ok so yes I can see the confusion here. I  was thinking of it more as a 'cancel changes' option.  i.e. i've done some changes and got confused, so i want to reload it from disk and start again.  So perhaps it's a 'cancel' button alongside the Save?  Maybe 'Cancel' buttons aren't very 'umbraco', and they assume a user would click away and back again to effect a cancel.  I'm don't feel strongly either way.  It was just nice to get a button working ;-)

Reply:
Ah ok, gotcha... "Revert to saved" (for want of a better phrase)

Reply:
I like Revert to saved too so have changed this :)